source/classes.tex

```diff
@@ -2473,7 +2473,6 @@
 \indextext{conversion!user-defined}%
 
 \pnum
```

Comment:
This is an odd (ineffective?) `\pnum` before a pure grammar block?

Reply:
Yes, this should have been deleted.

If you want to, I can fix that and force-push.

Reply:
fixed